src/store/drawing/__tests__/reducer-test.ts

```diff
@@ -20,6 +21,7 @@ describe('store/drawing/reducer', () => {
         location: 1,
         drawnPathGroups: [pathGroup1],
         stashedPathGroups: [pathGroup2],
+        stroke: { color: '#000', size: 1 },
```

Comment:
Should this be our "mock stroke" and reused for tests as needed?

Reply:
+1